settings/installer/ospd/overcloud/hostname/controllers.yml

```diff
@@ -0,0 +1,10 @@
+controller: 'yes'
```

Comment:
Same here, I think this should be part of the `role` rather than a user set param.
If a user wants to invoke this feature, it's a 'static' template that should reside under the role itself